.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+    "tabWidth": 4,
+    "semi": true,
```

Comment:
please stay close the currently implied settings

Reply:
Updated to implied settings.